core/unit_test/TestGraph.hpp

```diff
@@ -420,112 +420,244 @@ TEST_F(TEST_CATEGORY_FIXTURE(graph), empty_graph) {
   ex.fence();
 }
 
-template <typename ViewType>
-struct ForceGlobalLaunchFunctor {
+template <typename ViewType, size_t Size>
+struct SizedFunctor {
  public:
-  static constexpr size_t count =
-#if defined(KOKKOS_ENABLE_CUDA)
-      Kokkos::Impl::CudaTraits::ConstantMemoryUsage +
-#elif defined(KOKKOS_ENABLE_HIP)
-      Kokkos::Impl::HIPTraits::ConstantMemoryUsage +
-#endif
-      1;
+  static_assert(sizeof(ViewType) <= Size,
```

Comment:
```suggestion
  static_assert(sizeof(ViewType) < Size,
```
Zero-length array are enabled by compiler extension but it is not allowed in general.

Reply:
Indeed, and done.